### PR TITLE
added scripts to launch server on linux. 

### PR DESCRIPTION
**Pull request by INSANEMODE**
requires wine + [this](https://bugs.winehq.org/attachment.cgi?id=63662&action=diff) patch for bo2, or kernel version below 4.19 to work as of this commit.

### CODE REVIEW

**Comment by INSANEMODE** [comment]
image
tested and working on Ubuntu 20.04 using kernel 4.18.20.  tried to make it fairly similar to the .bat files.

File: !start_zm_server.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+echo -e '\033]2;'PlutoniumT6 - $NAME - Server restart'\007'  ## edits terminal tab title. may not work in all terminals
+echo "Visit plutonium.pw / Join the Discord (a6JM2Tv) for NEWS and Updates!"
+echo "Server "$NAME" will load $CFG and listen on port $PORT UDP!"
+echo "To shut down the server close this window first!"
+printf -v NOW '%(%F_%H:%M:%S)T' -1
+echo ""$NOW" $NAME server started."
+
+while true
+do
+wine start /wait /abovenormal z:$_mydir/t6rzm.exe -dedicated +sv_config $CFG +net_port $PORT  ## expects t6rzm.exe to be in same directory as this script
```
Comment (INSANEMODE) [inline !start_zm_server.sh:21]: if you are running headless/ over ssh you won't be able to start this.  please change this line to 
`xvfb-run wineconsole t6rzm.exe -dedicated +sv_config $CFG +net_port $PORT`

(make sure to install xfvb first)

File: !start_mp_server.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+echo -e '\033]2;'PlutoniumT6 - $NAME - Server restart'\007'  ## edits terminal tab title. may not work in all terminals
+echo "Visit plutonium.pw / Join the Discord (a6JM2Tv) for NEWS and Updates!"
+echo "Server "$NAME" will load $CFG and listen on port $PORT UDP!"
+echo "To shut down the server close this window first!"
+printf -v NOW '%(%F_%H:%M:%S)T' -1
+echo ""$NOW" $NAME server started."
+
+while true
+do
+wine start /wait /abovenormal z:$_mydir/t6rmp.exe -dedicated +sv_config $CFG +net_port $PORT  ## expects t6rzm.exe to be in same directory as this script
```
Comment (INSANEMODE) [inline !start_mp_server.sh]: if you are running headless/ over ssh you won't be able to start this. please change this line to
`xvfb-run wineconsole t6rmp.exe -dedicated +sv_config $CFG +net_port $PORT

(make sure to install xfvb first)

**Comment by INSANEMODE** [comment]
this is no longer needed.  can successfully run ```wineconsole !start_mp/zm_server.bat``` with no issue. and the server console works now.

**Comment by xerxes-at** [comment]
OK, I will close the PR. I wanted to take a closer look at it on the weekend.
For me running it via wine 5 worked out of the box thus why I wanted to do some tests first.

**Comment by marttoygoy5** [comment]
@INSANEMODE looks scary can you detail how to get it working on linux I want to do the same but wine is such a pain to configure

**Pull request closed without merging**